tests/cli/lib/lib.sh

```diff
@@ -1,5 +1,21 @@
 #!/usr/bin/env bash
 
+# Monkey-patch beakerlib to exit on first failure if COMPOSER_TEST_FAIL_FAST is
+# set. https://github.com/beakerlib/beakerlib/issues/42
+if [ -n "$COMPOSER_TEST_FAIL_FAST" ]; then
```

Comment:
Not a big deal but can we compare if this variable is == 1 or != 0. This will make it more stable in scenarios where you can be doing some debugging or wanting to disable this feature and set the value to anything but 1.

Reply:
Yeah, makes sense. I've pushed an update. Thank you.